diff --git a/bricks/_common/micropython.c b/bricks/_common/micropython.c
index 4fe922e..7b35b10 100644
--- a/bricks/_common/micropython.c
+++ b/bricks/_common/micropython.c
@@ -371,26 +371,26 @@ void pbsys_main_run_program(pbsys_main_program_t *program) {
     // Runs the requested downloaded or builtin user program.
     switch (program->id) {
 
-    #if PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_REPL
+        #if PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_REPL
         case PBIO_PYBRICKS_USER_PROGRAM_ID_REPL:
             // Run REPL with everything auto-imported.
             pb_package_pybricks_init(true);
             run_repl();
             break;
-    #endif
+        #endif
 
-    #if PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_PORT_VIEW && MICROPY_MODULE_FROZEN
+        #if PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_PORT_VIEW && MICROPY_MODULE_FROZEN
         case PBIO_PYBRICKS_USER_PROGRAM_ID_PORT_VIEW:
             pb_package_pybricks_init(false);
             pyexec_frozen_module("_builtin_port_view.py", false);
             break;
-    #endif
+        #endif
 
-    #if PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_IMU_CALIBRATION
+        #if PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_IMU_CALIBRATION
         case PBIO_PYBRICKS_USER_PROGRAM_ID_IMU_CALIBRATION:
             // Todo
             break;
-    #endif
+        #endif
 
         default:
             // Init Pybricks package without auto-import.
diff --git a/bricks/_common/mpconfigport.h b/bricks/_common/mpconfigport.h
index 172f438..9d347a7 100644
--- a/bricks/_common/mpconfigport.h
+++ b/bricks/_common/mpconfigport.h
@@ -26,15 +26,15 @@
 #define MICROPY_PY_SELECT                       (PYBRICKS_OPT_EXTRA_LEVEL1)
 
 #define MICROPY_PY_ERRNO_LIST \
-        X(EPERM) \
-        X(EIO) \
-        X(EBUSY) \
-        X(ENODEV) \
-        X(EINVAL) \
-        X(EOPNOTSUPP) \
-        X(EAGAIN) \
-        X(ETIMEDOUT) \
-        X(ECANCELED) \
+    X(EPERM) \
+    X(EIO) \
+    X(EBUSY) \
+    X(ENODEV) \
+    X(EINVAL) \
+    X(EOPNOTSUPP) \
+    X(EAGAIN) \
+    X(ETIMEDOUT) \
+    X(ECANCELED) \
 
 #if PYBRICKS_OPT_EXTRA_LEVEL2
 #define MICROPY_QSTR_BYTES_IN_HASH              (2)
@@ -156,31 +156,31 @@ typedef long mp_off_t;
 #endif
 
 #define MICROPY_VM_HOOK_LOOP \
-        do { \
-            PYBRICKS_VM_HOOK_LOOP_EXTRA \
-            extern bool pbio_os_run_processes_once(void); \
-            pbio_os_run_processes_once(); \
-        } while (0);
+    do { \
+        PYBRICKS_VM_HOOK_LOOP_EXTRA \
+        extern bool pbio_os_run_processes_once(void); \
+        pbio_os_run_processes_once(); \
+    } while (0);
 
 #define MICROPY_GC_HOOK_LOOP(i) do { \
-            if (((i) & 0xf) == 0) { \
-                MICROPY_VM_HOOK_LOOP \
-            } \
+        if (((i) & 0xf) == 0) { \
+            MICROPY_VM_HOOK_LOOP \
+        } \
 } while (0)
 
 #define MICROPY_INTERNAL_EVENT_HOOK \
-        do { \
-            PYBRICKS_VM_HOOK_LOOP_EXTRA \
-            extern bool pbio_os_run_processes_once(void); \
-            while (pbio_os_run_processes_once()) { \
-            } \
-        } while (0);
+    do { \
+        PYBRICKS_VM_HOOK_LOOP_EXTRA \
+        extern bool pbio_os_run_processes_once(void); \
+        while (pbio_os_run_processes_once()) { \
+        } \
+    } while (0);
 
 #define MICROPY_INTERNAL_WFE(TIMEOUT_MS) \
-        do { \
-            extern void pbio_os_run_processes_and_wait_for_event(void); \
-            pbio_os_run_processes_and_wait_for_event(); \
-        } while (0);
+    do { \
+        extern void pbio_os_run_processes_and_wait_for_event(void); \
+        pbio_os_run_processes_and_wait_for_event(); \
+    } while (0);
 
 // We need to provide a declaration/definition of alloca()
 #include <alloca.h>
diff --git a/bricks/virtualhub/mpconfigport.h b/bricks/virtualhub/mpconfigport.h
index 6bbb092..c92a180 100644
--- a/bricks/virtualhub/mpconfigport.h
+++ b/bricks/virtualhub/mpconfigport.h
@@ -80,10 +80,10 @@
 // advance the clock in those cases, which we mimic here by advancing the clock
 // every couple of MicroPython byte codes. This also polls to the event loop.
 #define PYBRICKS_VM_HOOK_LOOP_EXTRA \
-        do { \
-            extern void pbio_clock_test_advance_eventually(void); \
-            pbio_clock_test_advance_eventually(); \
-        } while (0);
+    do { \
+        extern void pbio_clock_test_advance_eventually(void); \
+        pbio_clock_test_advance_eventually(); \
+    } while (0);
 #else
 // When using the wall clock, time advances automatically but we still need to
 // request polling. This is done at the end of pbio_os_hook_wait_for_interrupt.
@@ -91,16 +91,16 @@
 // Instead of guessing with a number of instructions, here we can just poll
 // whenever the wall clock changes.
 #define PYBRICKS_VM_HOOK_LOOP_EXTRA \
-        do { \
-            static uint32_t clock_last; \
-            extern uint32_t pbdrv_clock_get_ms(void); \
-            uint32_t clock_now = pbdrv_clock_get_ms(); \
-            if (clock_last != clock_now) { \
-                extern void pbio_os_request_poll(void); \
-                pbio_os_request_poll(); \
-                clock_last = clock_now; \
-            } \
-        } while (0);
+    do { \
+        static uint32_t clock_last; \
+        extern uint32_t pbdrv_clock_get_ms(void); \
+        uint32_t clock_now = pbdrv_clock_get_ms(); \
+        if (clock_last != clock_now) { \
+            extern void pbio_os_request_poll(void); \
+            pbio_os_request_poll(); \
+            clock_last = clock_now; \
+        } \
+    } while (0);
 #endif
 
 // Allow printf for conventional purposes on native host, such as printing
diff --git a/lib/pbio/drv/block_device/block_device_ev3.c b/lib/pbio/drv/block_device/block_device_ev3.c
index 4fb69f3..6e67bea 100644
--- a/lib/pbio/drv/block_device/block_device_ev3.c
+++ b/lib/pbio/drv/block_device/block_device_ev3.c
@@ -710,14 +710,14 @@ static pbio_error_t pbdrv_block_device_write_disk(pbio_os_state_t *state, const
 //
 // NOTE: CSHOLD is *not* set here, so that CS is deasserted between each 16-bit unit
 #define MANUAL_ADC_CHANNEL(x)                                                       \
-        (1 << 26) |                                                                     \
-        (SPI_SPIDAT1_DFSEL_FORMAT1 << SPI_SPIDAT1_DFSEL_SHIFT) |                        \
-        (0 << (SPI_SPIDAT1_CSNR_SHIFT + PBDRV_EV3_SPI0_ADC_CS)) |                       \
-        (1 << (SPI_SPIDAT1_CSNR_SHIFT + PBDRV_EV3_SPI0_FLASH_CS)) |                     \
-        (1 << 12) |                                                                     \
-        (1 << 11) |                                                                     \
-        (((x) & 0xf) << 7) |                                                            \
-        (1 << 6)
+    (1 << 26) |                                                                     \
+    (SPI_SPIDAT1_DFSEL_FORMAT1 << SPI_SPIDAT1_DFSEL_SHIFT) |                        \
+    (0 << (SPI_SPIDAT1_CSNR_SHIFT + PBDRV_EV3_SPI0_ADC_CS)) |                       \
+    (1 << (SPI_SPIDAT1_CSNR_SHIFT + PBDRV_EV3_SPI0_FLASH_CS)) |                     \
+    (1 << 12) |                                                                     \
+    (1 << 11) |                                                                     \
+    (((x) & 0xf) << 7) |                                                            \
+    (1 << 6)
 
 #define PBDRV_ADC_EV3_NUM_DELAY_SAMPLES (2)
 
diff --git a/lib/pbio/drv/display/display_nxt.c b/lib/pbio/drv/display/display_nxt.c
index ebcb6b1..a0329b1 100644
--- a/lib/pbio/drv/display/display_nxt.c
+++ b/lib/pbio/drv/display/display_nxt.c
@@ -51,7 +51,7 @@
 #define SET_BIAS_POT0() (0x81)
 #define SET_BIAS_POT1(pot) (pot & 0xFF)
 #define SET_RAM_ADDR_CONTROL(auto_wrap, page_first, neg_inc, write_only_inc) \
-        (0x88 | (auto_wrap << 0) | (page_first << 1) |                    \
+    (0x88 | (auto_wrap << 0) | (page_first << 1) |                    \
     (neg_inc << 2) | (write_only_inc << 3))
 #define SET_ALL_PIXELS_ON(on) (0xA4 | (on & 1))
 #define SET_INVERSE_DISPLAY(on) (0xA6 | (on & 1))
diff --git a/lib/pbio/drv/gpio/gpio_ev3.c b/lib/pbio/drv/gpio/gpio_ev3.c
index 13610da..590022f 100644
--- a/lib/pbio/drv/gpio/gpio_ev3.c
+++ b/lib/pbio/drv/gpio/gpio_ev3.c
@@ -43,7 +43,7 @@ static void pbdrv_gpio_alt_gpio(const pbdrv_gpio_t *gpio) {
 // continuously and is not expected to block for too long,
 // only on the order of microseconds.
 #define PBDRV_GPIO_EV3_ARM_OWNS_GPIO_BANK(pin_index)    \
-        ((pin_index) > 32 || !pbdrv_rproc_is_ready())
+    ((pin_index) > 32 || !pbdrv_rproc_is_ready())
 
 static void gpio_write(const pbdrv_gpio_t *gpio, uint8_t value) {
     if (!gpio) {
diff --git a/lib/pbio/drv/gpio/gpio_ev3.h b/lib/pbio/drv/gpio/gpio_ev3.h
index 221531d..717bf69 100644
--- a/lib/pbio/drv/gpio/gpio_ev3.h
+++ b/lib/pbio/drv/gpio/gpio_ev3.h
@@ -30,15 +30,15 @@ typedef struct {
  * @param PIN The GPIO pin ID (zero-based).
  */
 #define PBDRV_GPIO_EV3_PIN(MUX, HIGH, LOW, BANK, PIN) \
-        { \
-            .bank = &(pbdrv_gpio_ev3_mux_t) { \
-                .mux_id = MUX, \
-                .mux_mask = SYSCFG_PINMUX##MUX##_PINMUX##MUX##_##HIGH##_##LOW, \
-                .mux_shift = SYSCFG_PINMUX##MUX##_PINMUX##MUX##_##HIGH##_##LOW##_SHIFT, \
-                .gpio_bank_id = BANK, \
-                .gpio_mode = SYSCFG_PINMUX##MUX##_PINMUX##MUX##_##HIGH##_##LOW##_GPIO##BANK##_##PIN, \
-            }, \
-            .pin = PIN, \
-        } \
+    { \
+        .bank = &(pbdrv_gpio_ev3_mux_t) { \
+            .mux_id = MUX, \
+            .mux_mask = SYSCFG_PINMUX##MUX##_PINMUX##MUX##_##HIGH##_##LOW, \
+            .mux_shift = SYSCFG_PINMUX##MUX##_PINMUX##MUX##_##HIGH##_##LOW##_SHIFT, \
+            .gpio_bank_id = BANK, \
+            .gpio_mode = SYSCFG_PINMUX##MUX##_PINMUX##MUX##_##HIGH##_##LOW##_GPIO##BANK##_##PIN, \
+        }, \
+        .pin = PIN, \
+    } \
 
 #endif // _INTERNAL_GPIO_TIAM1808_H_
diff --git a/lib/pbio/drv/pwm/pwm_lp50xx_stm32.c b/lib/pbio/drv/pwm/pwm_lp50xx_stm32.c
index 206da44..9b7e31a 100644
--- a/lib/pbio/drv/pwm/pwm_lp50xx_stm32.c
+++ b/lib/pbio/drv/pwm/pwm_lp50xx_stm32.c
@@ -72,7 +72,7 @@
 // REVISIT: this is for STM32F413 - may be different for other CPUs. There is
 // currently no equivelent of __LL_I2C_CONVERT_TIMINGS for FMPI2C in the STM32 HAL.
 #define FMPI2C_CONVERT_TIMINGS(PRESC, SCLDEL, SDADEL, SCLH, SCLL) \
-        (((PRESC) << FMPI2C_TIMINGR_PRESC_Pos) | \
+    (((PRESC) << FMPI2C_TIMINGR_PRESC_Pos) | \
     ((SCLDEL) << FMPI2C_TIMINGR_SCLDEL_Pos) | \
     ((SDADEL) << FMPI2C_TIMINGR_SDADEL_Pos) | \
     ((SCLH) << FMPI2C_TIMINGR_SCLH_Pos) | \
diff --git a/lib/pbio/drv/pwm/pwm_tlc5955_stm32.c b/lib/pbio/drv/pwm/pwm_tlc5955_stm32.c
index 842a63f..b869449 100644
--- a/lib/pbio/drv/pwm/pwm_tlc5955_stm32.c
+++ b/lib/pbio/drv/pwm/pwm_tlc5955_stm32.c
@@ -66,57 +66,57 @@ enum {
  * @param lsdvlt    LSD detection voltage selection (1-bit)
  */
 #define TLC5955_CONTROL_DATA(name, dc, mc, bc, dsprpt, tmgrst, rfresh, espwm, lsdvlt) \
-        uint8_t name[TLC5955_DATA_SIZE] = { \
-            /* bit 768 */      [0] = 1, \
-            /* bits 767-760 */ [1] = 0x96, \
-            /* bits 370-368 */ [50] = (((lsdvlt) << 2) | ((espwm) << 1) | (rfresh)) & 0xff, \
-            /* bits 367-360 */ [51] = (((tmgrst) << 7) | ((dsprpt) << 6) | ((bc) >> 1)) & 0xff, \
-            /* bits 359-352 */ [52] = (((bc) << 7) | ((bc) >> 0)) & 0xff, \
-            /* bits 351-344 */ [53] = (((bc) << 1) | ((mc) >> 2)) & 0xff, \
-            /* bits 343-336 */ [54] = (((mc) << 6) | ((mc) << 3) | (mc)) & 0xff, \
-            /* bits 335-328 */ [55] = (((dc) << 1) | ((dc) >> 6)) & 0xff, \
-            /* bits 327-320 */ [56] = (((dc) << 2) | ((dc) >> 5)) & 0xff, \
-            /* bits 319-312 */ [57] = (((dc) << 3) | ((dc) >> 4)) & 0xff, \
-            /* bits 311-304 */ [58] = (((dc) << 4) | ((dc) >> 3)) & 0xff, \
-            /* bits 303-296 */ [59] = (((dc) << 5) | ((dc) >> 2)) & 0xff, \
-            /* bits 295-288 */ [60] = (((dc) << 6) | ((dc) >> 1)) & 0xff, \
-            /* bits 287-280 */ [61] = (((dc) << 7) | ((dc) >> 0)) & 0xff, \
-            /* bits 279-272 */ [62] = (((dc) << 1) | ((dc) >> 6)) & 0xff, \
-            /* bits 271-264 */ [63] = (((dc) << 2) | ((dc) >> 5)) & 0xff, \
-            /* bits 263-256 */ [64] = (((dc) << 3) | ((dc) >> 4)) & 0xff, \
-            /* bits 255-248 */ [65] = (((dc) << 4) | ((dc) >> 3)) & 0xff, \
-            /* bits 247-240 */ [66] = (((dc) << 5) | ((dc) >> 2)) & 0xff, \
-            /* bits 239-232 */ [67] = (((dc) << 6) | ((dc) >> 1)) & 0xff, \
-            /* bits 231-224 */ [68] = (((dc) << 7) | ((dc) >> 0)) & 0xff, \
-            /* bits 223-216 */ [69] = (((dc) << 1) | ((dc) >> 6)) & 0xff, \
-            /* bits 215-208 */ [70] = (((dc) << 2) | ((dc) >> 5)) & 0xff, \
-            /* bits 207-200 */ [71] = (((dc) << 3) | ((dc) >> 4)) & 0xff, \
-            /* bits 199-192 */ [72] = (((dc) << 4) | ((dc) >> 3)) & 0xff, \
-            /* bits 191-184 */ [73] = (((dc) << 5) | ((dc) >> 2)) & 0xff, \
-            /* bits 183-176 */ [74] = (((dc) << 6) | ((dc) >> 1)) & 0xff, \
-            /* bits 175-168 */ [75] = (((dc) << 7) | ((dc) >> 0)) & 0xff, \
-            /* bits 167-160 */ [76] = (((dc) << 1) | ((dc) >> 6)) & 0xff, \
-            /* bits 159-152 */ [77] = (((dc) << 2) | ((dc) >> 5)) & 0xff, \
-            /* bits 151-144 */ [78] = (((dc) << 3) | ((dc) >> 4)) & 0xff, \
-            /* bits 143-136 */ [79] = (((dc) << 4) | ((dc) >> 3)) & 0xff, \
-            /* bits 135-128 */ [80] = (((dc) << 5) | ((dc) >> 2)) & 0xff, \
-            /* bits 127-120 */ [81] = (((dc) << 6) | ((dc) >> 1)) & 0xff, \
-            /* bits 119-112 */ [82] = (((dc) << 7) | ((dc) >> 0)) & 0xff, \
-            /* bits 111-104 */ [83] = (((dc) << 1) | ((dc) >> 6)) & 0xff, \
-            /* bits 103-96 */  [84] = (((dc) << 2) | ((dc) >> 5)) & 0xff, \
-            /* bits 95-88 */   [85] = (((dc) << 3) | ((dc) >> 4)) & 0xff, \
-            /* bits 87-80 */   [86] = (((dc) << 4) | ((dc) >> 3)) & 0xff, \
-            /* bits 79-72 */   [87] = (((dc) << 5) | ((dc) >> 2)) & 0xff, \
-            /* bits 71-64 */   [88] = (((dc) << 6) | ((dc) >> 1)) & 0xff, \
-            /* bits 63-56 */   [89] = (((dc) << 7) | ((dc) >> 0)) & 0xff, \
-            /* bits 55-48 */   [90] = (((dc) << 1) | ((dc) >> 6)) & 0xff, \
-            /* bits 47-40 */   [91] = (((dc) << 2) | ((dc) >> 5)) & 0xff, \
-            /* bits 39-32 */   [92] = (((dc) << 3) | ((dc) >> 4)) & 0xff, \
-            /* bits 31-24 */   [93] = (((dc) << 4) | ((dc) >> 3)) & 0xff, \
-            /* bits 23-16 */   [94] = (((dc) << 5) | ((dc) >> 2)) & 0xff, \
-            /* bits 15-8 */    [95] = (((dc) << 6) | ((dc) >> 1)) & 0xff, \
-            /* bits 7-0 */     [96] = (((dc) << 7) | ((dc) >> 0)) & 0xff, \
-    }
 
 typedef struct {
     /** HAL SPI data */
diff --git a/lib/pbio/drv/reset/reset_stm32.c b/lib/pbio/drv/reset/reset_stm32.c
index af2d60d..ed69409 100644
--- a/lib/pbio/drv/reset/reset_stm32.c
+++ b/lib/pbio/drv/reset/reset_stm32.c
@@ -43,14 +43,14 @@ void pbdrv_reset_init(void) {
 
 void pbdrv_reset(pbdrv_reset_action_t action) {
     switch (action) {
-        // Some platforms can't reboot in update mode. In those cases it will
-        // just shutdown instead so that update mode can be manually activated.
-        #if PBDRV_CONFIG_RESET_STM32_HAS_BLE_BOOTLOADER
+        // Some platforms can't reboot in update mode. In those cases it will
+        // just shutdown instead so that update mode can be manually activated.
+        #if PBDRV_CONFIG_RESET_STM32_HAS_BLE_BOOTLOADER
         case PBDRV_RESET_ACTION_RESET_IN_UPDATE_MODE:
             pbdrv_reset_stm32_bootloader_selector = BOOTLOADER_FIRMWARE_UPDATE_MODE;
             // fallthrough to PBDRV_RESET_ACTION_RESET
             __attribute__((fallthrough));
-        #endif // PBDRV_CONFIG_RESET_STM32_HAS_BLE_BOOTLOADER
+        #endif // PBDRV_CONFIG_RESET_STM32_HAS_BLE_BOOTLOADER
 
         case PBDRV_RESET_ACTION_RESET:
             NVIC_SystemReset(); // does not return
diff --git a/lib/pbio/drv/rproc/rproc_ev3_pru1.h b/lib/pbio/drv/rproc/rproc_ev3_pru1.h
index 83b217c..b9b1e5f 100644
--- a/lib/pbio/drv/rproc/rproc_ev3_pru1.h
+++ b/lib/pbio/drv/rproc/rproc_ev3_pru1.h
@@ -36,7 +36,7 @@ enum {
 };
 
 #define PBDRV_RPROC_EV3_PRU1_I2C_PACK_FLAGS(daddr, rlen, wlen, flags)   \
-        ((((daddr) & 0xff) << 24) |                                         \
+    ((((daddr) & 0xff) << 24) |                                         \
     (((rlen) & 0xff) << 16) |                                           \
     (((wlen) & 0xff) << 8) |                                            \
     ((flags) & 0xff))
diff --git a/lib/pbio/drv/uart/uart_ev3_pru_lib/cslr.h b/lib/pbio/drv/uart/uart_ev3_pru_lib/cslr.h
index f00558b..c17032d 100644
--- a/lib/pbio/drv/uart/uart_ev3_pru_lib/cslr.h
+++ b/lib/pbio/drv/uart/uart_ev3_pru_lib/cslr.h
@@ -18,40 +18,40 @@
 
 /* the Field MaKe macro */
 #define CSL_FMK(PER_REG_FIELD, val)                                         \
-        (((val) << CSL_##PER_REG_FIELD##_SHIFT) & CSL_##PER_REG_FIELD##_MASK)
+    (((val) << CSL_##PER_REG_FIELD##_SHIFT) & CSL_##PER_REG_FIELD##_MASK)
 
 /* the Field EXTract macro */
 #define CSL_FEXT(reg, PER_REG_FIELD)                                        \
-        (((reg) & CSL_##PER_REG_FIELD##_MASK) >> CSL_##PER_REG_FIELD##_SHIFT)
+    (((reg) & CSL_##PER_REG_FIELD##_MASK) >> CSL_##PER_REG_FIELD##_SHIFT)
 
 /* the Field INSert macro */
 #define CSL_FINS(reg, PER_REG_FIELD, val)                                   \
-        ((reg) = ((reg) & ~CSL_##PER_REG_FIELD##_MASK)                          \
+    ((reg) = ((reg) & ~CSL_##PER_REG_FIELD##_MASK)                          \
         | CSL_FMK(PER_REG_FIELD, val))
 
 /* the "token" macros */
 
 /* the Field MaKe (Token) macro */
 #define CSL_FMKT(PER_REG_FIELD, TOKEN)                                      \
-        CSL_FMK(PER_REG_FIELD, CSL_##PER_REG_FIELD##_##TOKEN)
+    CSL_FMK(PER_REG_FIELD, CSL_##PER_REG_FIELD##_##TOKEN)
 
 /* the Field INSert (Token) macro */
 #define CSL_FINST(reg, PER_REG_FIELD, TOKEN)                                \
-        CSL_FINS((reg), PER_REG_FIELD, CSL_##PER_REG_FIELD##_##TOKEN)
+    CSL_FINS((reg), PER_REG_FIELD, CSL_##PER_REG_FIELD##_##TOKEN)
 
 /* the "raw" macros */
 
 /* the Field MaKe (Raw) macro */
 #define CSL_FMKR(msb, lsb, val)                                             \
-        (((val) & ((1 << ((msb) - (lsb) + 1)) - 1)) << (lsb))
+    (((val) & ((1 << ((msb) - (lsb) + 1)) - 1)) << (lsb))
 
 /* the Field EXTract (Raw) macro */
 #define CSL_FEXTR(reg, msb, lsb)                                            \
-        (((reg) >> (lsb)) & ((1 << ((msb) - (lsb) + 1)) - 1))
+    (((reg) >> (lsb)) & ((1 << ((msb) - (lsb) + 1)) - 1))
 
 /* the Field INSert (Raw) macro */
 #define CSL_FINSR(reg, msb, lsb, val)                                       \
-        ((reg) = ((reg) & ~(((1 << ((msb) - (lsb) + 1)) - 1) << (lsb)))         \
+    ((reg) = ((reg) & ~(((1 << ((msb) - (lsb) + 1)) - 1) << (lsb)))         \
         | CSL_FMKR(msb, lsb, val))
 
 
diff --git a/lib/pbio/drv/usb/usb_ch9.h b/lib/pbio/drv/usb/usb_ch9.h
index b938193..dcc50f8 100644
--- a/lib/pbio/drv/usb/usb_ch9.h
+++ b/lib/pbio/drv/usb/usb_ch9.h
@@ -27,10 +27,10 @@
 //
 // The following macro helps construct these structs
 #define PBDRV_USB_TYPE_PUNNING_HELPER(type)                                     \
-        typedef union {                                                                 \
-            type##_t s;                                                                 \
-            uint32_t u[(sizeof(type##_t) + sizeof(uint32_t) - 1) / sizeof(uint32_t)];   \
-        } type##_union_t;
+    typedef union {                                                                 \
+        type##_t s;                                                                 \
+        uint32_t u[(sizeof(type##_t) + sizeof(uint32_t) - 1) / sizeof(uint32_t)];   \
+    } type##_union_t;
 
 // SETUP transaction packet structure
 typedef struct {
@@ -205,10 +205,10 @@ typedef struct PBDRV_PACKED {
 
 // {3408b638-09a9-47a0-8bfd-a0768815b665}
 #define USB_PLATFORM_CAP_GUID_WEBUSB    {                           \
-            .u0 = 0x3408b638,                                           \
-            .u1 = 0x09a9,                                               \
-            .u2 = 0x47a0,                                               \
-            .u3 = {0x8b, 0xfd, 0xa0, 0x76, 0x88, 0x15, 0xb6, 0x65},     \
+        .u0 = 0x3408b638,                                           \
+        .u1 = 0x09a9,                                               \
+        .u2 = 0x47a0,                                               \
+        .u3 = {0x8b, 0xfd, 0xa0, 0x76, 0x88, 0x15, 0xb6, 0x65},     \
 }
 
 typedef struct PBDRV_PACKED {
@@ -221,10 +221,10 @@ typedef struct PBDRV_PACKED {
 
 // {D8DD60DF-4589-4CC7-9CD2-659D9E648A9F}
 #define USB_PLATFORM_CAP_GUID_MS_20     {                           \
-            .u0 = 0xD8DD60DF,                                           \
-            .u1 = 0x4589,                                               \
-            .u2 = 0x4CC7,                                               \
-            .u3 = {0x9C, 0xD2, 0x65, 0x9D, 0x9E, 0x64, 0x8A, 0x9F},     \
+        .u0 = 0xD8DD60DF,                                           \
+        .u1 = 0x4589,                                               \
+        .u2 = 0x4CC7,                                               \
+        .u3 = {0x9C, 0xD2, 0x65, 0x9D, 0x9E, 0x64, 0x8A, 0x9F},     \
 }
 
 // These are not technically part of USB Chapter 9, but
diff --git a/lib/pbio/drv/usb/usb_common_desc.c b/lib/pbio/drv/usb/usb_common_desc.c
index 50b0b9a..edc3e28 100644
--- a/lib/pbio/drv/usb/usb_common_desc.c
+++ b/lib/pbio/drv/usb/usb_common_desc.c
@@ -30,8 +30,8 @@ const pbdrv_usb_webusb_url_desc_union_t pbdrv_usb_webusb_landing_page = {
 };
 
 #define MS_20_REGISTRY_DATA_EXTRA_SZ                    \
-        2 + sizeof(PBDRV_USB_MS_20_REG_PROPERTY_NAME) +     \
-        2 + sizeof(PBDRV_USB_MS_20_REG_PROPERTY_VAL)
+    2 + sizeof(PBDRV_USB_MS_20_REG_PROPERTY_NAME) +     \
+    2 + sizeof(PBDRV_USB_MS_20_REG_PROPERTY_VAL)
 
 const pbdrv_usb_ms_20_desc_set_union_t pbdrv_usb_ms_20_desc_set = {
     .s = {
diff --git a/lib/pbio/drv/usb/usb_ev3.c b/lib/pbio/drv/usb/usb_ev3.c
index 7d43b89..9b9d344 100644
--- a/lib/pbio/drv/usb/usb_ev3.c
+++ b/lib/pbio/drv/usb/usb_ev3.c
@@ -709,7 +709,7 @@ static void usb_device_intr(void) {
                     }
 
                     switch (setup_pkt.s.bRequest) {
-                    const char *s;
+                        const char *s;
 
                         case PBIO_PYBRICKS_USB_INTERFACE_READ_CHARACTERISTIC_GATT:
                             // Standard GATT characteristic
diff --git a/lib/pbio/include/pbio/light.h b/lib/pbio/include/pbio/light.h
index d9522eb..fdf53bc 100644
--- a/lib/pbio/include/pbio/light.h
+++ b/lib/pbio/include/pbio/light.h
@@ -26,7 +26,7 @@ typedef struct _pbio_color_light_t pbio_color_light_t;
  * @param [in]  value       The brightness (0 to 100)
  */
 #define PBIO_COLOR_LIGHT_ANIMATION_CELL(hue, saturation, value) \
-        { .h = (hue), .s = (saturation), .v = (value) }
+    { .h = (hue), .s = (saturation), .v = (value) }
 
 /** Sentinel value for a color light animation array. */
 #define PBIO_COLOR_LIGHT_ANIMATION_END_V (INT8_MAX)
diff --git a/lib/pbio/include/pbio/os.h b/lib/pbio/include/pbio/os.h
index d63477b..7ec8f58 100644
--- a/lib/pbio/include/pbio/os.h
+++ b/lib/pbio/include/pbio/os.h
@@ -198,12 +198,12 @@ struct _pbio_os_process_t {
  * @param [in]  condition   The condition (or statement expression) to check.
  */
 #define PBIO_OS_AWAIT_WHILE(state, condition)    \
-        do {                                         \
-            PBIO_OS_ASYNC_SET_CHECKPOINT(state);     \
-            if ((condition)) {                       \
-                return PBIO_ERROR_AGAIN;             \
-            }                                        \
-        } while (0)
+    do {                                         \
+        PBIO_OS_ASYNC_SET_CHECKPOINT(state);     \
+        if ((condition)) {                       \
+            return PBIO_ERROR_AGAIN;             \
+        }                                        \
+    } while (0)
 
 /**
  * Yields the protothread until the specified condition is true.
@@ -212,12 +212,12 @@ struct _pbio_os_process_t {
  * @param [in]  condition The condition (or statement expression) to check.
  */
 #define PBIO_OS_AWAIT_UNTIL(state, condition)    \
-        do {                                         \
-            PBIO_OS_ASYNC_SET_CHECKPOINT(state);     \
-            if (!(condition)) {                      \
-                return PBIO_ERROR_AGAIN;             \
-            }                                        \
-        } while (0)
+    do {                                         \
+        PBIO_OS_ASYNC_SET_CHECKPOINT(state);     \
+        if (!(condition)) {                      \
+            return PBIO_ERROR_AGAIN;             \
+        }                                        \
+    } while (0)
 
 /**
  * Awaits another (sub) protothread within the current (host) protothread until
@@ -228,13 +228,13 @@ struct _pbio_os_process_t {
  * @param [in]  calling_statement  The statement to await.
  */
 #define PBIO_OS_AWAIT(host_state, sub_state, calling_statement)  \
-        do {                                                         \
-            PBIO_OS_ASYNC_RESET((sub_state));                        \
-            PBIO_OS_ASYNC_SET_CHECKPOINT(host_state);                \
-            if ((calling_statement) == PBIO_ERROR_AGAIN) {           \
-                return PBIO_ERROR_AGAIN;                             \
-            }                                                        \
-        } while (0)
+    do {                                                         \
+        PBIO_OS_ASYNC_RESET((sub_state));                        \
+        PBIO_OS_ASYNC_SET_CHECKPOINT(host_state);                \
+        if ((calling_statement) == PBIO_ERROR_AGAIN) {           \
+            return PBIO_ERROR_AGAIN;                             \
+        }                                                        \
+    } while (0)
 
 
 /**
@@ -249,22 +249,22 @@ struct _pbio_os_process_t {
  * @param [in]  calling_statement_2    (Error-assigning) calling statement of the second sub protothread.
  */
 #define PBIO_OS_AWAIT_GATHER(host_state, err_1, sub_state_1, calling_statement_1, err_2, sub_state_2, calling_statement_2) \
-        do {                                                                                                   \
-            *err_1 = PBIO_ERROR_AGAIN;                                                                         \
-            *err_2 = PBIO_ERROR_AGAIN;                                                                         \
-            PBIO_OS_ASYNC_RESET((sub_state_1));                                                                \
-            PBIO_OS_ASYNC_RESET((sub_state_2));                                                                \
-            PBIO_OS_ASYNC_SET_CHECKPOINT(host_state);                                                          \
-            if (*err_1 == PBIO_ERROR_AGAIN) {                                                                  \
-                *err_1 = (calling_statement_1);                                                                \
-            }                                                                                                  \
-            if (*err_2 == PBIO_ERROR_AGAIN) {                                                                  \
-                *err_2 = (calling_statement_2);                                                                \
-            }                                                                                                  \
-            if (*err_1 == PBIO_ERROR_AGAIN || *err_2 == PBIO_ERROR_AGAIN) {                                    \
-                return PBIO_ERROR_AGAIN;                                                                       \
-            }                                                                                                  \
-        } while (0)
+    do {                                                                                                   \
+        *err_1 = PBIO_ERROR_AGAIN;                                                                         \
+        *err_2 = PBIO_ERROR_AGAIN;                                                                         \
+        PBIO_OS_ASYNC_RESET((sub_state_1));                                                                \
+        PBIO_OS_ASYNC_RESET((sub_state_2));                                                                \
+        PBIO_OS_ASYNC_SET_CHECKPOINT(host_state);                                                          \
+        if (*err_1 == PBIO_ERROR_AGAIN) {                                                                  \
+            *err_1 = (calling_statement_1);                                                                \
+        }                                                                                                  \
+        if (*err_2 == PBIO_ERROR_AGAIN) {                                                                  \
+            *err_2 = (calling_statement_2);                                                                \
+        }                                                                                                  \
+        if (*err_1 == PBIO_ERROR_AGAIN || *err_2 == PBIO_ERROR_AGAIN) {                                    \
+            return PBIO_ERROR_AGAIN;                                                                       \
+        }                                                                                                  \
+    } while (0)
 
 /**
  * Awaits two protothreads until one of them completes successfully or returns
@@ -277,14 +277,14 @@ struct _pbio_os_process_t {
  * @param [in]  calling_statement_2    (Error-assigning) calling statement of the second sub protothread.
  */
 #define PBIO_OS_AWAIT_RACE(host_state, sub_state_1, sub_state_2, calling_statement_1, calling_statement_2) \
-        do {                                                                                                   \
-            PBIO_OS_ASYNC_RESET((sub_state_1));                                                                \
-            PBIO_OS_ASYNC_RESET((sub_state_2));                                                                \
-            PBIO_OS_ASYNC_SET_CHECKPOINT(host_state);                                                          \
-            if ((calling_statement_1) == PBIO_ERROR_AGAIN && (calling_statement_2) == PBIO_ERROR_AGAIN) {      \
-                return PBIO_ERROR_AGAIN;                                                                       \
-            }                                                                                                  \
-        } while (0)
+    do {                                                                                                   \
+        PBIO_OS_ASYNC_RESET((sub_state_1));                                                                \
+        PBIO_OS_ASYNC_RESET((sub_state_2));                                                                \
+        PBIO_OS_ASYNC_SET_CHECKPOINT(host_state);                                                          \
+        if ((calling_statement_1) == PBIO_ERROR_AGAIN && (calling_statement_2) == PBIO_ERROR_AGAIN) {      \
+            return PBIO_ERROR_AGAIN;                                                                       \
+        }                                                                                                  \
+    } while (0)
 
 /**
  * Yields the protothread here once.
@@ -292,13 +292,13 @@ struct _pbio_os_process_t {
  * @param [in]  state     Protothread state.
  */
 #define PBIO_OS_AWAIT_ONCE(state)               \
-        do {                                        \
-            do_yield_now = 1;                       \
-            PBIO_OS_ASYNC_SET_CHECKPOINT(state);    \
-            if (do_yield_now) {                     \
-                return PBIO_ERROR_AGAIN;            \
-            }                                       \
-        } while (0)
+    do {                                        \
+        do_yield_now = 1;                       \
+        PBIO_OS_ASYNC_SET_CHECKPOINT(state);    \
+        if (do_yield_now) {                     \
+            return PBIO_ERROR_AGAIN;            \
+        }                                       \
+    } while (0)
 
 /**
  * Yields the protothread until the specified timer expires.
@@ -308,13 +308,13 @@ struct _pbio_os_process_t {
  * @param [in]  duration  The duration to wait for in milliseconds.
  */
 #define PBIO_OS_AWAIT_MS(state, timer, duration)      \
-        do {                                              \
-            pbio_os_timer_set(timer, duration);           \
-            PBIO_OS_ASYNC_SET_CHECKPOINT(state);          \
-            if (!pbio_os_timer_is_expired(timer)) {       \
-                return PBIO_ERROR_AGAIN;                  \
-            }                                             \
-        } while (0)                                       \
+    do {                                              \
+        pbio_os_timer_set(timer, duration);           \
+        PBIO_OS_ASYNC_SET_CHECKPOINT(state);          \
+        if (!pbio_os_timer_is_expired(timer)) {       \
+            return PBIO_ERROR_AGAIN;                  \
+        }                                             \
+    } while (0)                                       \
 
 void pbio_os_process_make_request(pbio_os_process_t *process, pbio_os_process_request_type_t request);
 
diff --git a/lib/pbio/include/pbio/protocol.h b/lib/pbio/include/pbio/protocol.h
index dd30855..e50a96e 100644
--- a/lib/pbio/include/pbio/protocol.h
+++ b/lib/pbio/include/pbio/protocol.h
@@ -35,9 +35,9 @@
  * Formatted as "X.Y.Z".
  */
 #define PBIO_PROTOCOL_VERSION_STR \
-        PBIO_XSTR(PBIO_PROTOCOL_VERSION_MAJOR) "." \
-        PBIO_XSTR(PBIO_PROTOCOL_VERSION_MINOR) "." \
-        PBIO_XSTR(PBIO_PROTOCOL_VERSION_PATCH)
+    PBIO_XSTR(PBIO_PROTOCOL_VERSION_MAJOR) "." \
+    PBIO_XSTR(PBIO_PROTOCOL_VERSION_MINOR) "." \
+    PBIO_XSTR(PBIO_PROTOCOL_VERSION_PATCH)
 
 
 /**
diff --git a/lib/pbio/include/pbio/version.h b/lib/pbio/include/pbio/version.h
index 0e4ba2f..68b8171 100644
--- a/lib/pbio/include/pbio/version.h
+++ b/lib/pbio/include/pbio/version.h
@@ -77,9 +77,9 @@
  * Format is "X.Y.Z" and may have an additional prerelease suffix.
  */
 #define PBIO_VERSION_STR \
-        PBIO_XSTR(PBIO_VERSION_MAJOR) "." \
-        PBIO_XSTR(PBIO_VERSION_MINOR) "."  \
-        PBIO_XSTR(PBIO_VERSION_MICRO) PBIO_VERSION_SUFFIX
+    PBIO_XSTR(PBIO_VERSION_MAJOR) "." \
+    PBIO_XSTR(PBIO_VERSION_MINOR) "."  \
+    PBIO_XSTR(PBIO_VERSION_MICRO) PBIO_VERSION_SUFFIX
 
 #endif // _PBIO_VERSION_H_
 
diff --git a/lib/pbio/src/image/image.c b/lib/pbio/src/image/image.c
index d6023d6..4b8bf7b 100644
--- a/lib/pbio/src/image/image.c
+++ b/lib/pbio/src/image/image.c
@@ -21,17 +21,17 @@
  * First coordinate must not be greater than the second one.
  */
 #define clip_or_return(c1, c2, l) \
-        do { \
-            if ((c1) >= (l) || (c2) <= 0) { \
-                return; \
-            } \
-            if ((c1) < 0) { \
-                (c1) = 0; \
-            } \
-            if ((c2) > (l)) { \
-                (c2) = (l); \
-            } \
-        } while (0)
+    do { \
+        if ((c1) >= (l) || (c2) <= 0) { \
+            return; \
+        } \
+        if ((c1) < 0) { \
+            (c1) = 0; \
+        } \
+        if ((c2) > (l)) { \
+            (c2) = (l); \
+        } \
+    } while (0)
 
 /**
  * Initialize an image, using external storage.
diff --git a/lib/pbio/sys/light.c b/lib/pbio/sys/light.c
index 99e53cd..5ab41a3 100644
--- a/lib/pbio/sys/light.c
+++ b/lib/pbio/sys/light.c
@@ -52,9 +52,9 @@ typedef struct {
 #define PBSYS_STATUS_LIGHT_DURATION_REPEAT (0)
 #define PBSYS_STATUS_LIGHT_DURATION_FOREVER (255)
 #define PBSYS_STATUS_LIGHT_INDICATION_PATTERN_REPEAT \
-        { .duration = PBSYS_STATUS_LIGHT_DURATION_REPEAT }
+    { .duration = PBSYS_STATUS_LIGHT_DURATION_REPEAT }
 #define PBSYS_STATUS_LIGHT_INDICATION_PATTERN_FOREVER(color_num) \
-        { .color = color_num, .duration = PBSYS_STATUS_LIGHT_DURATION_FOREVER, }
+    { .color = color_num, .duration = PBSYS_STATUS_LIGHT_DURATION_FOREVER, }
 
 static const pbsys_status_light_indication_pattern_element_t *const
 pbsys_status_light_indication_pattern_warning[] = {
diff --git a/lib/pbio/test/drv/test_bluetooth_btstack.c b/lib/pbio/test/drv/test_bluetooth_btstack.c
index e98c97e..411dca1 100644
--- a/lib/pbio/test/drv/test_bluetooth_btstack.c
+++ b/lib/pbio/test/drv/test_bluetooth_btstack.c
@@ -53,10 +53,10 @@ static void queue_packet(const uint8_t *buffer, uint16_t length) {
 }
 
 #define queue_command_complete(opcode, ...) {                       \
-            static const uint8_t result[] = {                           \
-                __VA_ARGS__                                             \
-            };                                                          \
-            _queue_command_complete(opcode, result, sizeof(result));    \
+        static const uint8_t result[] = {                           \
+            __VA_ARGS__                                             \
+        };                                                          \
+        _queue_command_complete(opcode, result, sizeof(result));    \
 }
 
 static void _queue_command_complete(uint16_t opcode, const uint8_t *result, uint16_t length) {
diff --git a/lib/pbio/test/src/test_image.c b/lib/pbio/test/src/test_image.c
index c364bdc..8328082 100644
--- a/lib/pbio/test/src/test_image.c
+++ b/lib/pbio/test/src/test_image.c
@@ -122,7 +122,7 @@ static void test_image_dump(void) {
 }
 
 #define tt_want_clipping_correct() \
-        tt_want_(test_image_clipping_correct(), "want(clipping correct)", \
+    tt_want_(test_image_clipping_correct(), "want(clipping correct)", \
     test_image_dump())
 
 static void test_image_prepare_small_image(pbio_image_t *small) {
@@ -157,7 +157,7 @@ static void test_image_small_diff(const char *expected) {
 }
 
 #define tt_want_small_image(expected) \
-        tt_want_(test_image_small_equal((expected)), "want(image match)", \
+    tt_want_(test_image_small_equal((expected)), "want(image match)", \
     test_image_small_diff((expected)))
 
 static void test_image_fill(void *env) {
diff --git a/lib/pbio/test/src/test_light_matrix.c b/lib/pbio/test/src/test_light_matrix.c
index f54afe6..4c9f53e 100644
--- a/lib/pbio/test/src/test_light_matrix.c
+++ b/lib/pbio/test/src/test_light_matrix.c
@@ -24,7 +24,7 @@
 #define IMAGE_DATA(...) (const uint8_t[DATA_SIZE]) {__VA_ARGS__}
 
 #define tt_want_light_matrix_data(...) \
-        tt_want_int_op(memcmp(test_light_matrix_set_pixel_last_brightness, \
+    tt_want_int_op(memcmp(test_light_matrix_set_pixel_last_brightness, \
     (const uint8_t[DATA_SIZE]) {__VA_ARGS__}, DATA_SIZE), ==, 0)
 
 static const uint8_t test_animation[] = {
diff --git a/lib/pbio/test/src/test_lump.c b/lib/pbio/test/src/test_lump.c
index 4c1c274..7f4c5cc 100644
--- a/lib/pbio/test/src/test_lump.c
+++ b/lib/pbio/test/src/test_lump.c
@@ -25,7 +25,7 @@
 // TODO: submit this upstream
 #ifndef tt_want_float_op
 #define tt_want_float_op(a, op, b) \
-        tt_assert_test_type(a, b,#a " "#op " "#b, float, (val1_ op val2_), "%f", (void)0)
+    tt_assert_test_type(a, b,#a " "#op " "#b, float, (val1_ op val2_), "%f", (void)0)
 #endif
 
 struct _pbdrv_uart_dev_t {
@@ -99,13 +99,13 @@ end:
 }
 
 #define SIMULATE_RX_MSG(msg) do { \
-            PBIO_OS_AWAIT(state, &child, err = simulate_rx_msg(&child, (msg), PBIO_ARRAY_SIZE(msg))); \
-            tt_assert_msg(err == PBIO_SUCCESS, #msg); \
+        PBIO_OS_AWAIT(state, &child, err = simulate_rx_msg(&child, (msg), PBIO_ARRAY_SIZE(msg))); \
+        tt_assert_msg(err == PBIO_SUCCESS, #msg); \
 } while (0)
 
 #define SIMULATE_TX_MSG(msg) do { \
-            PBIO_OS_AWAIT(state, &child, err = simulate_tx_msg(&child, (msg), PBIO_ARRAY_SIZE(msg))); \
-            tt_assert_msg(err == PBIO_SUCCESS, #msg); \
+        PBIO_OS_AWAIT(state, &child, err = simulate_tx_msg(&child, (msg), PBIO_ARRAY_SIZE(msg))); \
+        tt_assert_msg(err == PBIO_SUCCESS, #msg); \
 } while (0)
 
 static const uint8_t msg_speed_115200[] = { 0x52, 0x00, 0xC2, 0x01, 0x00, 0x6E }; // SPEED 115200
diff --git a/lib/pbio/test/test-pbio.h b/lib/pbio/test/test-pbio.h
index cc075ff..206124a 100644
--- a/lib/pbio/test/test-pbio.h
+++ b/lib/pbio/test/test-pbio.h
@@ -14,11 +14,11 @@
 
 // Use this macro to define tests that _don't_ require the pbio/os event loop
 #define PBIO_TEST(name) \
-        { #name, name, TT_FORK, NULL, NULL }
+    { #name, name, TT_FORK, NULL, NULL }
 
 // Use this macro to define tests that _do_ require the pbio/os event loop
 #define PBIO_THREAD_TEST(name) \
-        { #name, pbio_test_run_thread, TT_FORK, &pbio_test_setup, name }
+    { #name, pbio_test_run_thread, TT_FORK, &pbio_test_setup, name }
 
 void pbio_test_run_thread(void *env);
 
diff --git a/pybricks/common/pb_type_device.h b/pybricks/common/pb_type_device.h
index c5c8722..cf9e988 100644
--- a/pybricks/common/pb_type_device.h
+++ b/pybricks/common/pb_type_device.h
@@ -52,8 +52,8 @@ extern const mp_obj_type_t pb_type_device_method;
  * to the mapping function called later, when the data is available.
  */
 #define PB_DEFINE_CONST_TYPE_DEVICE_METHOD_OBJ(obj_name, mode_id, get_values_func) \
-        const pb_type_device_method_obj_t obj_name = \
-        {{&pb_type_device_method}, .mode = mode_id, .get_values = get_values_func}
+    const pb_type_device_method_obj_t obj_name = \
+    {{&pb_type_device_method}, .mode = mode_id, .get_values = get_values_func}
 
 mp_obj_t pb_type_device_method_call(mp_obj_t self_in, size_t n_args, size_t n_kw, const mp_obj_t *args);
 mp_obj_t pb_type_pupdevices_method(mp_obj_t self_in, size_t n_args, size_t n_kw, const mp_obj_t *args);
diff --git a/pybricks/util_mp/pb_kwarg_helper.h b/pybricks/util_mp/pb_kwarg_helper.h
index 743987d..1802c83 100644
--- a/pybricks/util_mp/pb_kwarg_helper.h
+++ b/pybricks/util_mp/pb_kwarg_helper.h
@@ -25,7 +25,7 @@
 #define FI_9(WHAT, N, X, ...) WHAT(N - 9, X) FI_8(WHAT, N, __VA_ARGS__)
 #define GET_MACRO(_1, _2, _3, _4, _5, _6, _7, _8, _9, NAME, ...) NAME
 #define PB_FOR_EACH_IDX(action, ...) \
-        GET_MACRO(__VA_ARGS__, FI_9, FI_8, FI_7, FI_6, FI_5, FI_4, FI_3, FI_2, FI_1, )(action, NUM_ARGS(__VA_ARGS__), __VA_ARGS__)
+    GET_MACRO(__VA_ARGS__, FI_9, FI_8, FI_7, FI_6, FI_5, FI_4, FI_3, FI_2, FI_1, )(action, NUM_ARGS(__VA_ARGS__), __VA_ARGS__)
 
 // Make a QSTR, even if the name is generated from a macro
 #define MAKE_QSTR_(name) MP_QSTR_##name
@@ -34,8 +34,8 @@
 // Parse given positional and keyword arguments against a list of allowed arguments
 // First n_ignore arguments are required arguments for which no keyword can be given.
 #define PB_PARSE_ARGS(parsed_args, n_args, pos_args, kw_args, allowed_args, n_ignore) \
-        mp_arg_val_t parsed_args[MP_ARRAY_SIZE(allowed_args)]; \
-        mp_arg_parse_all(n_args - n_ignore, pos_args + n_ignore, kw_args, MP_ARRAY_SIZE(allowed_args), allowed_args, parsed_args)
+    mp_arg_val_t parsed_args[MP_ARRAY_SIZE(allowed_args)]; \
+    mp_arg_parse_all(n_args - n_ignore, pos_args + n_ignore, kw_args, MP_ARRAY_SIZE(allowed_args), allowed_args, parsed_args)
 
 // The following functions make use of the aforementioned PB_PARSE_ARGS macro, but they first
 // auto-generate the allowed_args table to simplify notation in the pybricks modules.
@@ -43,9 +43,9 @@
 // Generate table entry from an argument name, its requirements, and its default value
 #define GET_ARG_NAME(name, required, value) name##_in
 #define GET_ARG_SPEC(name, required, value) { \
-            .qst = MAKE_QSTR(name), \
-            .flags = required, \
-            .defval = value, \
+        .qst = MAKE_QSTR(name), \
+        .flags = required, \
+        .defval = value, \
 }
 
 // Unpack a table entry into three arguments
@@ -56,28 +56,28 @@
 
 // Create the arguments table, parse it, and declare mp_obj_t's for each one
 #define PB_PARSE_GENERIC(n_args, pos_args, kw_args, n_ignore, ...) static const mp_arg_t allowed_args[] = { \
-            PB_FOR_EACH_IDX(PB_ARG_DO, __VA_ARGS__) \
+        PB_FOR_EACH_IDX(PB_ARG_DO, __VA_ARGS__) \
 }; \
-        PB_PARSE_ARGS(parsed_args, n_args, pos_args, kw_args, allowed_args, n_ignore); \
-        PB_FOR_EACH_IDX(GEN_ARG_OBJ, __VA_ARGS__)
+    PB_PARSE_ARGS(parsed_args, n_args, pos_args, kw_args, allowed_args, n_ignore); \
+    PB_FOR_EACH_IDX(GEN_ARG_OBJ, __VA_ARGS__)
 // Parse the arguments of a function
 #define PB_PARSE_ARGS_FUNCTION(n_args, pos_args, kw_args, ...) \
-        PB_PARSE_GENERIC(n_args, pos_args, kw_args, 0, __VA_ARGS__)
+    PB_PARSE_GENERIC(n_args, pos_args, kw_args, 0, __VA_ARGS__)
 
 // Parse the arguments of a class method, skipping self
 #define PB_PARSE_ARGS_METHOD_SKIP_SELF(n_args, pos_args, kw_args, ...) \
-        PB_PARSE_GENERIC(n_args, pos_args, kw_args, 1, __VA_ARGS__)
+    PB_PARSE_GENERIC(n_args, pos_args, kw_args, 1, __VA_ARGS__)
 
 // Parse the arguments of a class method, including self
 #define PB_PARSE_ARGS_METHOD(n_args, pos_args, kw_args, self_type, self_name, ...) \
-        PB_PARSE_GENERIC(n_args, pos_args, kw_args, 1, __VA_ARGS__) \
-        self_type *self_name = MP_OBJ_TO_PTR(pos_args[0]);
+    PB_PARSE_GENERIC(n_args, pos_args, kw_args, 1, __VA_ARGS__) \
+    self_type *self_name = MP_OBJ_TO_PTR(pos_args[0]);
 
 // Parse the arguments of a __init__ (without parsing self)
 #define PB_PARSE_ARGS_CLASS(n_args, n_kw, pos_and_kw_args, ...) \
-        mp_map_t kw_args; \
-        mp_map_init_fixed_table(&kw_args, n_kw, pos_and_kw_args + n_args); \
-        PB_PARSE_GENERIC(n_args, pos_and_kw_args, &kw_args, 0, __VA_ARGS__)
+    mp_map_t kw_args; \
+    mp_map_init_fixed_table(&kw_args, n_kw, pos_and_kw_args + n_args); \
+    PB_PARSE_GENERIC(n_args, pos_and_kw_args, &kw_args, 0, __VA_ARGS__)
 
 // Required argument
 #define PB_ARG_REQUIRED(name) (name, MP_ARG_OBJ | MP_ARG_REQUIRED, { })
diff --git a/pybricks/util_mp/pb_obj_helper.h b/pybricks/util_mp/pb_obj_helper.h
index b34964d..a65333f 100644
--- a/pybricks/util_mp/pb_obj_helper.h
+++ b/pybricks/util_mp/pb_obj_helper.h
@@ -60,8 +60,8 @@ typedef struct {
 
 // Generic entry of the attributes dictionary.
 #define PB_DEFINE_CONST_ATTR_RO(name_, type, field) {           \
-            .name = (name_),                                        \
-            .offset = offsetof(type, field),                        \
+        .name = (name_),                                        \
+        .offset = offsetof(type, field),                        \
 }
 
 // Attribute handler for any object that has an attribute dictionary assigned